### PR TITLE
QRF Branding

### PR DESCRIPTION
**Pull request by DQuraini**
Updating the certificates for LMS and the header logo for CMS.

### CODE REVIEW

File: cms/static/sass/elements/_header.scss [hunk 1]
```diff
@@ -118,7 +118,7 @@
       display: block;
 
       img {
-        max-height: ($baseline*2);
+        max-height: ($baseline*2.4);
```
Comment (Salomari1987) [inline cms/static/sass/elements/_header.scss]: @DQuraini we never do changes inline in the sass files in edx-platform, please move this change to the theme repo and I will approve it from there

**Comment by DQuraini** [comment]
@Salomari1987 the style was intended for studio. I have added a new scss file called it extras for styling.

File: cms/static/sass/_build.scss [hunk 2]
```diff
@@ -81,3 +81,4 @@
 
 @import 'developer'; // used for any developer-created scss that needs further polish/refactoring
 @import 'shame';     // used for any bad-form/orphaned scss
+@import 'extras'; // extra styles for edraak
```
Comment (Salomari1987) [inline cms/static/sass/_build.scss]: I like your approach, just one final tweak. Remove **extras**, then go to directory: `common/static/sass/edraak`, there you will find a file called `_arabic-mathjax.scss`, next to it and in the same directory, add a file called `_header.scss`, then come back here, and under the line where `arabic-mathjax` is imported, import the header file. This should work and I will approve and merge.

**Comment by DQuraini** [comment]
@Salomari1987 done

**Pull request merged**